### PR TITLE
Feat/heat 344 create deploy env workflow

### PR DESCRIPTION
**Pull request by Sandhya1874**

### CODE REVIEW

File: .github/actions/build-test-and-deploy/cloud-platform-deploy/action.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    description: The KUBE_NAMESPACE
+    required: true
+  token:
+    description: The KUBE_TOKEN
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v3
```
Comment (james-jdgtl) [inline .github/actions/build-test-and-deploy/cloud-platform-deploy/action.yml:30]: I think @v3 is deprecated.. can you bump to @v4?

**Review by james-jdgtl — COMMENTED** [review]
just a minor one for the checkout@v3 line

File: .github/actions/build-test-and-deploy/cloud-platform-deploy/action.yml (hunk 1 above)
Comment (Sandhya1874) [reply .github/actions/build-test-and-deploy/cloud-platform-deploy/action.yml]: Done

**Review by james-jdgtl — APPROVED** [review]
looks good!

**Pull request merged**